README.md

```diff
@@ -1,2 +1,199 @@
-# react-capstone-project
-In this project will be React - Redux using Redux toolkit by build a project movie rating application along with Axios for API calls, React Router DOM for Routing and Node-SASS for our SCSS compile to CSS.
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
```

Comment:
[OPTIONAL] Since you have changed your API and theme of the application since the previous review. 

- Kindly update your README and make the project name and description relevant to your project.
- Please remove the Microverse logo and heading. It is meant to be replaced by your project logo if there's any.
- Please make sure that there are no broken links on your README. Please fix the link to your license.